specification/search/data-plane/Azure.Search/preview/2021-04-01-Preview/searchservice.json

```diff
@@ -0,0 +1,9480 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "SearchServiceClient",
+    "description": "Client that can be used to manage and query indexes and documents, as well as manage other resources, on a search service.",
+    "version": "2021-04-01-Preview",
```

Comment:
Just thought I would point out the API version in the backend is actually 2021-04-**30**-Preview, not 2021-04-**01**-Preview. Looks like you would need to change that in quite a few places for this PR instead the folder name.